src/sentry/api/serializers/models/group.py

```diff
@@ -179,6 +182,24 @@ def __init__(
         self.collapse = collapse
         self.expand = expand
 
+    def _serialize_assigness(
```

Comment:
This is semi faithful move of `ActorTuple.resolve_dict` which was a massive misnomer.  That method was only referenced here, nowhere else, and what it *actually* does is try to return a dictionary of group_ids -> serializable entities.  This new method captures most of that logic but uses the new user_service.serialize_users rather than expect raw Users which implicitly get serialized.  Keeping it in this serializer class also makes clear its specific, non generic intent.